site/pages/badger-profile/index.js

```diff
@@ -27,7 +27,7 @@ const BadgerProfile = ({ badger }: { badger: Badger }) => {
       <div className={styles.profileContainer}>
         <div className={styles.profilePictureContainer}>
           <img
-            src={badger.imageUrl}
+            src={badger.secondaryImageUrl || badger.primaryImageUrl}
```

Comment:
Should it be maybe secondary image or placeholder? Will layout work with primary image when it is substituting for secondary?